### PR TITLE
feat(ailments): add version check package

### PR DESCRIPTION
**Pull request by rdlabo**
I made common version check method, and add version check package.
package: ionic/core, ionic/schematics-angular, angular/cli, angular-devkit/core, angular-devkit/schematics

### CODE REVIEW

File: packages/@ionic/cli-utils/src/lib/project/angular/ailments.ts [hunk 1]
```diff
@@ -8,11 +8,77 @@ import { pkgFromRegistry, pkgManagerArgs } from '../../utils/npm';
...
+    pkgName: '@ionic/angular',
+    treatmentVisitURL: ['https://github.com/ionic-team/ionic/releases'],
+  }));
+  registry.register(new MajorUpdateAvailable(deps, {
+    id: 'ionic-for-angular-major-update-available',
+    pkgName: '@ionic/angular',
+    treatmentVisitURL: ['https://blog.ionicframework.com', 'https://github.com/ionic-team/ionic/releases'],
+  }));
+
+  // @ionic/core
```
Comment (dwieeb) [inline packages/@ionic/cli-utils/src/lib/project/angular/ailments.ts]: `@ionic/core` won't be in the list of dependencies for starters in the stable release, so this check isn't needed

File: packages/@ionic/cli-utils/src/lib/project/angular/ailments.ts [hunk 2]
```diff
@@ -23,34 +89,36 @@ export interface AutomaticallyTreatableAngularAilmentDeps extends AutomaticallyT
...
 
-class IonicForAngularUpdateAvailable extends AngularAilment {
-  id = 'ionic-for-angular-update-available';
-  currentVersion?: string;
-  latestVersion?: string;
+    let boldTreatmentVisitURL: string[] = [];
+    this.pkgParams.treatmentVisitURL.forEach((url: string) => {
+      boldTreatmentVisitURL = boldTreatmentVisitURL.concat(chalk.bold(url));
+    });
+    this.pkgParams.treatmentVisitURL = boldTreatmentVisitURL;
```
Comment (dwieeb) [inline packages/@ionic/cli-utils/src/lib/project/angular/ailments.ts]: Try something like this:

```ts
this.pkgParams.treatmentVisitURL = this.pkgParams.treatmentVisitURL.map(url => chalk.bold(url));
```

**Comment by dwieeb** [comment]
@rdlabo This is great work. 👍

**Comment by rdlabo** [comment]
thanks for review. It changed! (I forgot map function...)

**Comment by dwieeb** [comment]
Awesome, looks great

**Pull request merged**